request_verification_api.py

```diff
@@ -0,0 +1,32 @@
+import requests
+from dotenv import load_dotenv
+import os
+
+load_dotenv(dotenv_path='./credentials.env')
+
+
+REDIRECT_URL = 'http://127.0.0.1:5000/results'
+CLIENT_ID = os.environ.get('CLIENT_ID')
+CLIENT_SECRET = os.environ.get('CLIENT_SECRET')
+
+
+request_verification_url = "https://api.app.authenteq.com/web-idv/verification-url?redirectUrl={}".format(REDIRECT_URL)
+
+if (CLIENT_ID or CLIENT_SECRET) is None:
+    print("Create .env file in the root of the project with CLIENT_ID and CLIENT_SECRET variables. You will find their"
```

Comment:
Update the message to reflect the fact that those vars should be in environment variables not in file.